wetterdienst/settings.py

```diff
@@ -36,29 +31,29 @@ class Settings:
         "cache_disable": False,
         "cache_dir": platformdirs.user_cache_dir(appname="wetterdienst"),
         "fsspec_client_kwargs": {},
-        "humanize": True,
-        "tidy": True,
-        "si_units": True,
-        "skip_empty": False,
-        "skip_threshold": 0.95,
-        "skip_criteria": "min",
-        "dropna": False,
-        "interp_use_nearby_station_until_km": 1,
+        "scalar_humanize": True,
+        "scalar_shape": "long",
+        "scalar_si_units": True,
+        "scalar_skip_empty": False,
+        "scalar_skip_threshold": 0.95,
+        "scalar_skip_criteria": "min",
+        "scalar_dropna": False,
+        "scalar_interpolation_use_nearby_station_distance": 1,
     }
 
     def __init__(
         self,
         cache_disable: Optional[bool] = None,
         cache_dir: Optional[pathlib.Path] = None,
         fsspec_client_kwargs: Optional[dict] = None,
-        humanize: Optional[bool] = None,
-        tidy: Optional[bool] = None,
-        si_units: Optional[bool] = None,
-        skip_empty: Optional[bool] = None,
-        skip_threshold: Optional[float] = None,
-        skip_criteria: Optional[Literal["min", "mean", "max"]] = None,
-        dropna: Optional[bool] = None,
-        interp_use_nearby_station_until_km: Optional[Union[float, int]] = None,
+        scalar_humanize: Optional[bool] = None,
+        scalar_shape: Optional[Literal["wide", "long"]] = None,
+        scalar_si_units: Optional[bool] = None,
+        scalar_skip_empty: Optional[bool] = None,
+        scalar_skip_threshold: Optional[float] = None,
+        scalar_skip_criteria: Optional[Literal["min", "mean", "max"]] = None,
+        scalar_dropna: Optional[bool] = None,
+        scalar_interpolation_use_nearby_station_distance: Optional[Union[float, int]] = None,
```

Comment:
Do we really need to prefix all those with `scalar_`? I think it becomes a bit noisy throughout the code.

Reply:
This came to my mind when changing `tidy` to `format` because how would the user know what format refers to?

Reply:
I see. But now that it is called `shape`, it is unambiguous again, right? So we can get along without prefixing?

Reply:
When there is really need for grouping, I am not against it. We did it slightly at [^1], for the `socket_` options. I am sure similar large amounts of options are present in the pandas code base as well, but I did not experience this kind of grouping before, where half of the options have a prefix for whatever reason.

P.S.: What does `scalar` mean anyway? See also my next comment about "naming things".

[^1]: https://github.com/crate/crate-python/blob/0.30.1/src/crate/client/connection.py#L31-L51

Reply:
We could name it point instead as well.

Reply:
> I see. But now that it is called `shape`, I think it is reasonably unambiguous again, right? So we can get along without prefixing?

I'd like the idea that we copy basically the names from the env variables (there we need to use prefixes to not mess up with other envs), but we could also drop it again. Does it hurt the user anyway to use `scalar_/point_/ts_` as prefix?

Reply:
> `Scalar` would refer to the `ScalarRequest` class but maybe `PointRequestCore` and `PointSettings` would make even more sense.

Honestly, I never got accustomed to the naming of the `ScalarRequest` class ;].

> We could name it point or timeseries instead as well.

Wouldn't "Observation" be a better name for describing the "group of data" it is actually handling?

> I like the idea that we copy basically the names from the env variables.

I fully agree. Having a flat list of setting identifiers is fortunate, so we need a prefix.

> Does it hurt the user anyway to use `scalar_`/`point_`/`ts_` as prefix?

It is fine. But let's think about the proper name. And, I think a shorter prefix than `scalar_` would be sweet. I like `ts_`, but what about `obs_`?

Reply:
Ok, I recognize that "observation" would mismatch the "MOSMIX" type of data.

Reply:
I think I am gravitating towards "time series data", or "Zeitseriendaten", as an umbrella term, effectively using the `ts_` prefix for designating corresponding symbols. What do you think?

Reply:
then let's name it timeseries (`TimeSeriesRequest`, `TimeSeriesValues`)

Reply:
I think not applying camel case to `Series` will be fine. `TimeseriesRequest` and `TimeseriesValues` looks much better to me.